docs/site/tutorials/core/7-observation.md

```diff
@@ -0,0 +1,51 @@
+---
+lang: en
+title: 'Observation of life cycle events'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/core-tutorial-part7.html
+---
+
+There are life cycle events associated with an application, typically `start`
+and `stop`.
+[Life cycle observers](https://loopback.io/doc/en/lb4/Life-cycle.html) allows
+various types of artifacts to participate in the application life cycles.
+
+You can make use of the
+[life cycle observer generator](https://loopback.io/doc/en/lb4/Life-cycle-observer-generator.html)
+to create observers easily.
+
+## How is cache maintained
+
+In the
+[CachingService](https://github.com/strongloop/loopback-next/blob/master/examples/greeting-app/src/caching-service.ts),
+there is a time-to-live (ttl) setting for each cache items. That means when the
+cache item is expired, it will be removed from the cache.
+
+The life cycle observer provides a way to look at the in-memory caching as part
+of the application life cycle and remove the ones that are expired. See the
+[cache.observer.ts](https://github.com/strongloop/loopback-next/blob/master/examples/greeting-app/src/observers/cache.observer.ts)
+as an example.
+
+## Updating cache ttl at runtime
+
+During the start of the application, the observer calls CachingService.start
```

Comment:
wrap CachingService.start ?

Reply:
hm.. isn't it the same?  The observer does call the CachingService.start() function.

Reply:
https://github.com/strongloop/loopback-next/blob/master/examples/greeting-app/src/observers/cache.observer.ts#L25-L27

Reply:
I mean CachingService.start -> `CachingService.start`

Either way is good to me~

Reply:
fixed. thanks.